src/main/resources/config/specs/default-topics-spec.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
it's huge confusing that same file mean different things. in all ms its config to dynamic lep consumers but in this ms, it's config for internal handler